readthedocs/settings/docker_compose.py

```diff
@@ -66,7 +66,11 @@ def RTD_EXT_THEME_DEV_SERVER(self):
     @property
     def RTD_EMBED_API_EXTERNAL_DOMAINS(self):
         domains = super().RTD_EMBED_API_EXTERNAL_DOMAINS
-        domains.append(r'.*\.readthedocs\.io')
+        domains.extend([
+            r'.*\.readthedocs\.io',
+            r'.*\.org\.readthedocs\.build',
+            r'.*\.com\.readthedocs\.build',
+        ])
```

Comment:
Should this include rtd-hosted.com as well?

Reply:
Yeap!